lib/local-db.spec.ts

```diff
@@ -0,0 +1,115 @@
+import fs from 'fs';
+import { LocalDb } from './local-db';
+
+jest.mock('fs');
+
+describe('LocalDatabase', () => {
+  describe('Create', () => {
+    it('should create db', () => {
+      jest.spyOn(fs, 'existsSync').mockReturnValueOnce(false);
+      LocalDb.create();
+
+      expect(fs.copyFileSync).toBeCalled();
+    });
+
+    it('should not create db', () => {
+      jest.spyOn(fs, 'existsSync').mockReturnValueOnce(true);
+      LocalDb.create();
+
+      expect(fs.copyFileSync).not.toBeCalled();
+    });
+  });
+
+  describe('Get', () => {
+    it('should get an item', () => {
+      jest.spyOn(fs, 'readFileSync').mockReturnValueOnce('[{ "id": "123" }, { "id": "345" }]');
+
+      expect(LocalDb.get({ Key: { id: '123' } })).toStrictEqual({ id: '123' });
+    });
+
+    it('should not get an item', () => {
+      jest.spyOn(fs, 'readFileSync').mockReturnValueOnce('[{ "id": "123" }, { "id": "345" }]');
+
+      expect(LocalDb.get({ Key: { id: '000' } })).toStrictEqual(undefined);
+    });
+
```

Comment:
test it with an empty database as well, to make sure it doesn't generate an black hole. You never know
```
jest.spyOn(fs, 'readFileSync').mockReturnValueOnce('[]');
```